Makefile

```diff
@@ -51,6 +51,15 @@ watch-unit:
 watch-compile:
 	$(BIN_DIR)/tsc --watch  --noEmit --skipLibCheck
 
+e2e:
+	yarn build && \
+	yarn test:e2e
+	
+e2e-in-ci:
+	. ./.envrc && \
+	yarn build && \
```

Comment:
Why do we need to `build` here?

We are `build`ing in the `integration-in-ci` task because we are also test-executing the cronjob (which relies on the code having been compiled).
Is there a dependency in this job on code that was compiled?